main/test/ca/uwaterloo/flix/library/TestFoldable.flix

```diff
@@ -190,6 +190,22 @@ mod TestFoldable {
         let xs = 1 :: 2 :: 3 :: 4 :: 5 :: Nil;
         runLogger(Foldable.foldRightM(foldRightStep, 0, xs)) == (5 :: 4 :: 3 :: 2 :: 1 :: Nil, 15)
 
+    /////////////////////////////////////////////////////////////////////////////
+    // length                                                                //
+    /////////////////////////////////////////////////////////////////////////////
+
+    @test
+    def length01(): Bool =
```

Comment:
Probably should add these tests just before the tests for count, since that is where you placed `length`.

Reply:
Unless I am mistaken (very possible), it seems like tests for ```count``` (as well as those for ```sum```, ```sumWith```, and ```forall``` are not present in TestFoldable.flix). The next function present in the testing file is ```head```, whose tests are at the very top.

Reply:
Hmm! We should add those then!

Reply:
Maybe you can do that in a subsequent PR.